apps/condo/domains/miniapp/schema/B2CAppProperty.js

```diff
@@ -2,13 +2,13 @@
  * Generated by `createschema miniapp.B2CAppProperty 'app:Relationship:B2CApp:PROTECT; address:Text;' --force`
  */
 
-const { Text, Relationship } = require('@keystonejs/fields')
+const { Relationship } = require('@keystonejs/fields')
```

Comment:
Can you please replace field with string representation to be KS6 compatible?
`type: Relationship` -> `type: 'Relationship'`